### PR TITLE
Get return code and GLIBC dockerfile fix 

### PR DESCRIPTION
**Pull request by paigerube14**
Had hit issue like [this one](https://issues.redhat.com/browse/OCPBUGS-36420), had updated the prow job to not use the dockerfile. With this change, I'm hoping to actually use the dockerfile here instead of [configuring myself in prow ](https://github.com/openshift/release/blob/ceb87bf36c310dc7863577d2eee2e1767578a0f3/ci-operator/config/openshift-qe/ocpqe-security-tools/openshift-qe-ocpqe-security-tools-main__dast.yaml#L6-L22)

Adding proper return code if a high alert is found, see passing job of new return code here: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/ssml/254/console

https://issues.redhat.com/browse/OCPQE-23853

### CODE REVIEW

**Comment by paigerube14** [comment]
@asood-rh are you able to take a look at this PR

File: dast/deploy_ssml_api.sh [hunk 1]
```diff
@@ -64,10 +64,12 @@ for api_doc in ${API_URL_LIST}; do
   oc delete pvc rapidast-pvc
   (( counter++ ))
 done
-
 python find_alert_types.py
-
+high_alert_status=$(echo $?)
+echo "high alert return status $high_alert_status"
 if [ $phase != "Succeeded" ]; then
```
Comment (asood-rh) [inline dast/deploy_ssml_api.sh:70]: @paigerube14 Does it make sense to call find_alert_types.py to process results only if phase == succeeded.

Comment (paigerube14) [reply dast/deploy_ssml_api.sh:70]: Not necessarily, if there are multiple apis defined the $phase would only be for the last one so we would still want totry to get the alerts from the others

**Pull request merged**